go-controller/pkg/ovn/gress_policy.go

```diff
@@ -477,7 +477,6 @@ func (gp *gressPolicy) destroy() error {
 		if err := gp.peerAddressSet.Destroy(); err != nil {
 			return err
 		}
-		gp.peerAddressSet = nil
```

Comment:
seems OK because this destroy call will end up in teh entire policy getting deleted, so nothing should ever touch this peerAddressSet again.

Reply:
@aojea or @JacobTanenbaum can you also PTAL at this. Just want to make sure we dont miss some implication of not setting this to nil anymore.

Reply:
I don't think there is an issue. Personally I would like it there just in case/for completeness but that is a small nit and no need to reformat the PR. It is one of the last parts of deleting a network policy